src/components/visualizations/headline/ResponsiveText.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { debounce, isNumber } from 'lodash';
+
+export default class ResponsiveText extends React.PureComponent {
```

Comment:
Delete this file it's already part of goodstrap lib

Reply:
Coming soon in future comits